website/content/docs/import/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Secrets import
+description: Secrets import allows you to safely onboard secrets from external sources into Vault KV for management.
+---
+
+
+# Secrets import
+
+<EnterpriseAlert product="vault" />
+
+@include 'alerts/beta.mdx'
+
+Having collections of secrets across various external systems can present several challenges to an organization, such as
+increased operational overhead, a greater risk of secrets sprawl, and more. Vault can offer a single source of truth (SSOT)
+solution to managing these secrets, providing increased security and reducing that overhead, yet the effort required
+to migrate all your secrets from multiple external sources into Vault can be steep. To aid in this onboarding process,
+secrets import provides a mechanism to automate it. 
+
+A secret that is imported from an external system is stored in Vault's KVv2 Secrets Engine. The import process is defined 
+an import plan, which is an HCL file. Using HCL enables users to declaratively configure which of their secrets will be read
+from an external source and where exactly they will be stored in Vault.
+
+There are three main types of HCL blocks used to configure a secrets import plan. They are source blocks, the Vault destination
+block, and mapping blocks.
+
+## Vault destinations
+
+Secrets that get imported are stored into a Vault KVv2 Secrets Engine mount. The destination will accept the mount path of
+the desired KVv2 mount and optionally a namespace. This specifies the exact location to store the secrets in. A destination
+is defined by an HCL block in the format of `destination_vault`.
+
+### HCL syntax
+
+#### Argument reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Vault destination block used to internally reference this destination inside of a mapping.
+
+* `address` - (Optional) The address of the Vault server with the KVv2 Secrets Engine for which the secrets will be imported into.
+
+* `token` - (Optional) A token for authentication to the Vault server located at the specified address.
+
+* `namespace` - (Optional) The name of the Vault namespace containing the specified KVv2 mount.
+
+* `mount` - (Optional) The name of the KVv2 mount in Vault for which the secrets will be imported into.
+
+#### Example
+
+An example `destination_vault` block may look like the following:
+
+```hcl
+destination_vault {
+  name = "my-dest-1"
+  address = "http://127.0.0.1:8200/"
+  token = "root"
+  namespace = "ns-1"
+  mount = "mount-1"
+}
+```
```

Comment:
Same comment here, equal signs aligned & address + token should not be specified.